docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,256 @@
+Use "Cloud" `none` for on-premise setups.
+Field "Region" is not required to be set for "Cloud" `none`.
+If the cloud, region or distribution don't exist yet in the Commodore defaults repo, you can create the following files:
+
+* For a missing cloud, create file `cloud/<cloudname>.yml` in the repo.
+  If the missing cloud has multiple regions, also create file `cloud/<cloudname>/params.yml` in the repo.
+  For multi-region clouds the file `cloud/<cloudname>.yml` file in the repo should always have exactly the following content:
++
+.<cloud>/cloudname.yml
+[source,yaml]
+----
+classes:
+  - global.cloud.<cloudname>.params
+  - global.cloud.<cloudname>.${facts:region}
+----
+
+* For a missing region, or if you're creating a new multi-region cloud, create file `cloud/<cloudname>/<regionname>.yml` in the repo.
+* For a missing distribution, create file `distribution/<distributionname>.yml` in the repo.
+
+.. "Rancher Cluster ID (Fact: rancher_cluster_id)": If it's a Rancher cluster, add the Cluster ID from the Rancher Management Server to this field
+.. "Repository Type": Leave at "auto" to have Lieutenant manage the cluster catalog repo on https://git.vshn.net
+.. "GitRepo URL": Leave empty if "Repository Type" is auto.
+   Check with the {projectsyn-ig}, if you have special requirements for the cluster catalog repo.
+
++
+[NOTE]
+====
+Not all combinations work smoothly.
+If in doubt, seek help from the {projectsyn-ig}.
+====
+
+. Add initial configuration in the `<cluster-id>.yml` file in the tenant's Project Syn configuration repository (often called the "Syn tenant repo").
+See {portal-lieutenant-tenants} for a list of tenants and their associated configuration repositories.
+
+. Run Commodore to generate the initial cluster catalog.
++
+Grab the Kubeconfig for the {synfra-cluster}
+Then you can setup the environment for Commodore as follows:
++
+[source,bash]
+----
+# Adjust this to point to where you've stored the Kubeconfig file for the Synfra cluster
+export KUBECONFIG=/path/to/synfra.kubeconfig
+export COMMODORE_API_TOKEN=$(kubectl config view -o jsonpath='{.users[?(@.name == "syn-synfra")].user.token}'  --raw)
+export COMMODORE_API_URL="https://api.syn.vshn.net"
+----
++
+Now you can follow the steps in the https://syn.tools/syn/how-tos/compile-catalog.html#_compilation[Project Syn "Compile a catalog" how-to]
+
+. Add required tokens to Vault.
+Check the documentation for the K8s distribution of your cluster for required secrets.
++
+All distributions require tokens for https://k8up.io[K8up] and the https://github.com/projectsyn/component-cluster-backup[cluster-backup] component. Make sure you've got the https://www.vaultproject.io/docs/install[Vault CLI] available locally and run the following commands:
++
+[source,bash]
```

Comment:
Might be simplified by https://ticket.vshn.net/browse/PORTAL-10, not sure if we should push that one forward.

Reply:
A reference to PORTAL-10 would be fine here as well.